desc/integrals/bounce_integral.py

```diff
@@ -289,11 +292,11 @@ def check_points(self, z1, z2, pitch_inv, *, plot=True, **kwargs):
     def integrate(
         self,
         integrand,
+        points,
```

Comment:
should this be an optional argument, with the default to calculate from `pitch_inv`?

Reply:
I'd rather user just call bounce.points()

Reply:
The reason for this is just to avoid duplicate calculations right?

I figure there are 2 main use cases for this:
1. in some compute or objective function where the user doesn't interact directly, and we as developers will know to precompute the bounce points and pass them in
2. Used directly by a user to just compute some arbitary bounce integral. In that case I'm not a fan of having multiple arguments to a function that have to be self-consistent (and where checking self-consistency is non-trivial), so in that case I think it makes more sense to allow the user to just specify the pitch values

Reply:
I have moved `points` to the final allowable nameless argument so that users can use the function as if the parameter wasn't there